src/index.js

```diff
@@ -11,14 +11,17 @@ var addDollarSign = function(value) {
 }
 
 exports.toCents = function(dollarString) {
-  if(typeof(dollarString) !== 'string') {
-    return dollarString;
+  if (!exports.valid(dollarString)) {
+    return 0;
```

Comment:
I prefer @nguyenchr's idea. `null` is better than `-1` is better than `0`. But I wonder if it should throw an exception if it's being asked to do something (convert to cents, in this case) that it can't.